pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -101,6 +101,10 @@ func validateBrokerSpec(spec *sc.BrokerSpec, fldPath *field.Path) field.ErrorLis
 
 	}
 
+	if spec.InsecureSkipTLSVerify && len(spec.CABundle) > 0 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("CABundle"), spec.CABundle, "CABundle cannot be used when InsecureSkipTLSVerify is true"))
```

Comment:
The JSON field name is `caBundle` (and the other is `insecureSkipTLSVerify`), so it should be:
```go
allErrs = append(allErrs, field.Invalid(fldPath.Child("caBundle"), spec.CABundle, "caBundle cannot be used when insecureSkipTLSVerify is true"))
```